messages/messages.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
```

Comment:
we can make Kres build this: 

https://github.com/cosi-project/runtime/blob/master/.kres.yaml

Reply:
Done, I think.